2021/21xxx/CVE-2021-21510.json

```diff
@@ -1,17 +1,70 @@
 {
```

Comment:
'dell.com/support/kbdoc/en-us/000183758/dsa-2021-041-dell-emc-idrac-8-security-update-for-a-host-header-injection-vulnerability' does not match '^(ftp|http)s?://\\S+$'